src/shapes/Text/Text.ts

```diff
@@ -63,20 +69,16 @@ type TPathAlign = 'baseline' | 'center' | 'ascender' | 'descender';
  * needs the the info of previous graphemes already filled
  * Override to customize measuring
  */
-export type GraphemeBBox<onPath = false> = {
+export type GraphemeBBox = {
```

Comment:
this was too hard to be flexible ( or simply i can't use it )
I m thinking that maybe we just keep al the properties required and we just add the extra 3 everywhere.